src/forms/dropdown/Dropdown.tsx

```diff
@@ -20,26 +20,40 @@ import {STOP_PROPAGATION} from ".";
 
 /**
  * DropDown Props
- * @param {label} label of checkbox
+ * @param {label} label of dropdown
  * @param {isNested} property to nest dropdown
  * @param {className} css property
  * @param {button} dropdown button
  * @param {showCaret} caret property
+ * @param {direction} direction to open a menu
```

Comment:
Small suggestion - More than 'direction to open menu' it should be 'position of the dropdown'

Reply:
We have used same naming convention in tooltip and signpost so keeping this as it is.